### PR TITLE
Add landing page for setting Sourcegraph up at hackathons

### PR DESCRIPTION
**Pull request by attfarhan**
A page we can send people to if they want to set Sourcegraph up during a hackathon. We'll provide access to Enterprise features to set them up for a successful demo.

image

### CODE REVIEW

File: website/src/pages/hackathons.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                        <meta property="og:title" content="Set up Sourcegraph at a Hackathon" />
+                    </Helmet>
+                    <div className="pricing-bg" />
+                    <div className="jumbotron text-right dark-7">
+                        <div className="row">
+                            <div className="sales py-0 d-flex flex-grow flex-column flex-lg-row align-items-center justify-content-around">
+                                <div>
+                                    <h1 className="text-center">Set up Sourcegraph during a hackathon</h1>
+                                    <p className="text-center measure">
+                                        Hackathons are the perfect time to set up code search and intelligence for your
```
Comment (dadlerj) [inline website/src/pages/hackathons.tsx]: I'd just cut down on this section — feels a bit long for a subtitle (unless this is our standard)

File: website/src/pages/hackathons.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <section className="d-flex justify-content-around flex-column flex-lg-row sales bg-white">
+                        <div className="panel-area mr-lg-5 mt-0 mb-2">
+                            <div className="d-flex flex-column panel">
+                                <p>
+                                    Want to bring lasting value to your company during a hackathon? Set up Sourcegraph
```
Comment (dadlerj) [inline website/src/pages/hackathons.tsx]: This is great, but I've never heard this wording before. It sounds a bit serious/formal to me ("lasting value") — maybe something fun/flippant like "Want to win your Hackathon?"

File: website/src/pages/hackathons.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                                    and bring the power of code search and code intelligence to your engineering team.
+                                </p>
+                                <ul>
+                                    <li>
+                                        We'll give you <b>access to our Enterprise features</b> so you can ship
```
Comment (dadlerj) [inline website/src/pages/hackathons.tsx]: Shorten these bullets, e.g. (whatever wording you want is fine!)
* We'll give you a trial license key for all of our [Enterprise features](/pricing)
* We'll give you live tech support
* We'll ship you a bag of stickers, shirts, socks, and other great swag!

File: website/src/pages/hackathons.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                                        Sourcegraph's full feature set to your team and for your demo.
+                                    </li>
+                                    <li>
+                                        We'll <b>answer any questions and share tips</b> to make sure you can
+                                        successfully deploy and demo Sourcegraph
+                                    </li>
+                                </ul>
+
+                                <p>Fill out the form, and we'll get back to you ASAP on how to get started!</p>
```
Comment (dadlerj) [inline website/src/pages/hackathons.tsx]: How about "fill out the form or Tweet at us" to make it a bit less formal? This page simply existing is the main thing that'll help :)

**Comment by attfarhan** [comment]
Thanks for the review @dadlerj. I'm not sure what else we can offer, but we can add on to this if we come up with ideas. Also, totally agree that it would be awesome to get case studies on past hackathon successes. Would make this page even more effective.

**Pull request merged**